packages/amazon/src/serverGroup/configure/wizard/instanceType/advancedMode/common/InstanceTypeTable.tsx

```diff
@@ -0,0 +1,131 @@
+import _ from 'lodash';
+import React from 'react';
+import { arrayMove, SortEnd } from 'react-sortable-hoc';
+
+import { CpuCreditsToggle } from '../../CpuCreditsToggle';
+import { AWSProviderSettings } from '../../../../../../aws.settings';
+import { CustomHeading, CustomTableBody, CustomTableFooter, CustomTableHeader } from '../custom/index';
+import { IAmazonInstanceTypeCategory } from '../../../../../../instance/awsInstanceType.service';
+import { NonCustomHeading, NonCustomTableBody, NonCustomTableHeader } from '../nonCustom/index';
+import { IAmazonInstanceTypeOverride } from '../../../../serverGroupConfiguration.service';
+
+import '../advancedMode.less';
+
+export interface IInstanceTypeTableProps {
+  currentProfile: string;
+  selectedInstanceTypesMap: Map<string, IAmazonInstanceTypeOverride>;
+  unlimitedCpuCreditsInCmd: boolean;
+  profileDetails: IAmazonInstanceTypeCategory; // todo: useMemo pdk
+  availableInstanceTypesList: string[]; // todo: useMemo pdk
+  handleInstanceTypesChange: (instanceTypes: IAmazonInstanceTypeOverride[]) => void;
+  setUnlimitedCpuCredits: (unlimitedCpuCredits: boolean | undefined) => void;
+}
+
+export function InstanceTypeTable(props: IInstanceTypeTableProps) {
```

Comment:
I would take a look at the `Table` components in deck. `MangedResourceHistoryModal` has a good example of using the table layouts and a custom row. 

I think it would remove the need for custom vs non-custom headers, etc.

Reply:
I'll echo that i think the table code here could probably be simplified.  I'm not sure if the `Table` component will help here as it was purpose built for a specific screen and the API can be a bit klunky.